### ISSUE TITLE
NSIS: main argv conversion failed. Used to work fine. builder:wine v1.2.1 -> v1.3.2

### ISSUE THREAD

**Issue by MaciejKrawczyk** [electron-userland/electron-builder#2591]
* **19.34.2**: 

* **Windows**: 

Hello. My app was building just fine about 2 months ago. I created a new app out of it (I rebranded it) and suddenly it didn't work. I thought that it was the cause. But then I took the exactly same build from the previous app and only updated the version number and pushed it to GitHub, to see that it also fails. So I made virtually no changes to that code, yet it fails now. 

Both builds were using electron-builder v19.34.2 and nsis 3.0.1.13 Electron version changed from 1.6.15 to 1.8.2 though.

This is the successful log from 2 months ago: https://travis-ci.org/ContentOS/contentos-desktop-app/jobs/311462534

And this one is from now: https://travis-ci.org/ContentOS/contentos-desktop-app/jobs/342318965

As you can see, I made no significant changes at all: https://github.com/ContentOS/contentos-desktop-app/commit/1a7c51c2b320815ab5bda189b0296cf8bf124ce6 

Only some changes in CSS and updated the version number. So the fault is most likely in newer version of Electron used.

Any help appreciated. And big thanks for making this tool.

**Comment by MaciejKrawczyk** [comment]
Looks like it's not Electron version's fault. I downgraded to 1.6.15 (set it in the config) and it still fails: https://travis-ci.org/ContentOS/contentos-desktop-app/jobs/342330766

But now I noticed that the successful build used electronuserland/builder:wine v1.2.1
The newer failing build uses v1.3.2 So this is most likely the problem. Is it possible to downgrade the image somehow, so I could test if the older one still works?
